lesson-4/script.js

```diff
@@ -0,0 +1,74 @@
+const correctResult = (0.1 + 0.2).toFixed(1);
+console.log(correctResult);
+const userLogin = prompt("Enter your login.");
+console.log(userLogin);
+const userPassword = prompt("Enter your password.");
+console.log(userPassword);
+if (userLogin == "admin" && userPassword == "12pass33210") {
+    console.log("Access is allowed.");
+} else if (userLogin != "admin" && userPassword != "12pass33210") {
+    console.log("This user have no access to the panel.");
+} else {
+    console.log("Login or password are wrong.")
+}
+let userMoney = +prompt("How much money do you have?");
+console.log(`User have ${userMoney} UAH`);
+console.log("Here is our price list below:")
+let Watermelon = 30;
+console.log(`Watermelon: ${Watermelon} UAH`);
+let Carrot = 8;
+console.log(`Carrot: ${Carrot} UAH`);
+let Potato = 12;
```

Comment:
```suggestion
const potato = 12;
```

or even better `const pricePotato`